types/error/MaxResourceLimitExceededError.raml

```diff
@@ -0,0 +1,8 @@
+#%RAML 1.0 DataType
+(package): Error
+type: ErrorObject
+displayName: MaxResourceLimitExceededError
+discriminatorValue: MaxResourceLimitExceeded
+properties:
+  exceededResource:
+    type: ReferenceTypeId
```

Comment:
in the docs is pointing to the ReferenceType so is it correct to put as a type the Id? it doesn't work with ReferenceType

Reply:
It's only named differently in the API ref, so it's fine